test/prob/gamma/gamma_ccdf_log_test.hpp

```diff
@@ -90,10 +90,10 @@ class AgradCcdfLogGamma : public AgradCcdfLogTest {
   stan::return_type_t<T_y, T_shape, T_inv_scale> ccdf_log_function(
       const T_y& y, const T_shape& alpha, const T_inv_scale& beta, const T3&,
       const T4&, const T5&) {
-    using boost::math::gamma_q;
-    using stan::math::gamma_q;
-    using std::log;
+    using boost::math::gamma_p;
+    using stan::math::gamma_p;
+    using stan::math::log1m;
 
-    return log(gamma_q(alpha, beta * y));
+    return log1m(gamma_p(alpha, beta * y));
```

Comment:
Doing log1m(cdf) worked better than log(ccdf) for forward mode. Check here for the lcdf: https://github.com/stan-dev/math/blob/develop/test/prob/gamma/gamma_cdf_log_test.hpp#L84